examples/production/values.yaml

```diff
@@ -37,7 +21,43 @@ spire-server:
       drop: [ALL]
     seccompProfile:
       type: RuntimeDefault
+
+  topologySpreadConstraints:
+    - maxSkew: 1
+      topologyKey: topology.kubernetes.io/zone
+      whenUnsatisfiable: DoNotSchedule
+      labelSelector:
+        matchLabels:
+          app.kubernetes.io/name: server
+          app.kubernetes.io/instance: spire
+
+  resources:
+    requests:
+      cpu: 200m
+      memory: 128Mi
+    limits:
+      cpu: 1000m
+      memory: 1Gi
+
+  dataStorage:
+    enabled: true
+    size: 10Gi
+    accessMode: ReadWriteOnce
+    storageClass: gp3-encrypted
+
+  jwtIssuer: &jwtIssuer oidc-spire.eu-west-1-marco-test.my-domain.com
+  ca_subject:
+    country: "NL"
+    organization: "Marco Home Office"
+    common_name: spire.eu-west-1-marco-test.my-domain.com
```

Comment:
Should probably leave the jwtIssuer, ca_subject, clusterName and trustDomain alone?